scene/main/scene_tree.cpp

```diff
@@ -1545,10 +1545,16 @@ Ref<SceneTreeTimer> SceneTree::create_timer(double p_delay_sec, bool p_process_a
 Ref<Tween> SceneTree::create_tween() {
 	_THREAD_SAFE_METHOD_
 	Ref<Tween> tween = memnew(Tween(true));
+	tween->parent_tree = this;
 	tweens.push_back(tween);
 	return tween;
 }
 
+bool SceneTree::remove_tween(Ref<Tween> &p_tween) {
+	_THREAD_SAFE_METHOD_
+	return tweens.erase(p_tween);
```

Comment:
`tweens` is a linked List, so erasing can be costly.

Reply:
Thanks for pointing that out! That may be a good reason to allow users to manually create Tween objects, and then add them to the SceneTree (or not) as necessary, rather than only allowing them to be created via `create_tween`.